release-notes/8.0/preview/api-diff/preview4/Microsoft.WindowsDesktop.App/8.0-preview4_System.Media.md

```diff
@@ -0,0 +1,10 @@
+# System.Media
+
+``` diff
+ namespace System.Media {
+     public class SoundPlayer : Component, ISerializable {
+         protected SoundPlayer(SerializationInfo serializationInfo, StreamingContext context);
```

Comment:
this wasn't added, it just got obsoleted in .net 8. either the attribute should be what's shown, or it shouldn't show up.